src/main/java/fish/payara/extras/diagnostics/asadmin/CollectAsadmin.java

```diff
@@ -70,6 +70,9 @@ public class CollectAsadmin extends BaseAsadmin {
     @Param(name = ParamConstants.DOMAIN_XML_PARAM, optional = true, defaultValue = "true")
     private boolean collectDomainXml;
 
+    @Param(name = ParamConstants.OBFUSCATE_DOMAIN_XML_PARAM, optional = true, defaultValue = "true")
```

Comment:
As per ticket description "An option should be added to the collect-diagnostics command: --`obfuscate` true|false " but the parameter name is currently `obfuscateDomainXml`. Please update the parameter name to match the description for consistency.